### PR TITLE
Docker registry sync

### PR DESCRIPTION
**Pull request by etsauer**

### CODE REVIEW

**Comment by etsauer** [comment]
@oybed @JayKayy please review

File: docker-registry-sync [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+usage() {
+  echo "$0:
+
+Description: Syncs images from a public docker registry to a private registry. Use this to populate private registries in a closed off environment.
+Must be run from a linux host capable of running docker commands which has access both to the internet and the private registry.
+
```
Comment (oybed) [inline docker-registry-sync:8]: Maybe include a comment about the format of the "image-file" - i.e.: one entry per line, etc.

**Comment by oybed** [comment]
@etsauer looks good to me - see comment about adding some info re: input file.

@JayKayy I'll let you have the final word  / merge if you're OK with the implementation

**Comment by etsauer** [comment]
@oybed Enhanced usage output.

**Comment by oybed** [comment]
@etsauer thumbs up from me. :-) :+1:

**Pull request merged**